src/foam/net/node/StaticFileHandler.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.net.node',
+  name: 'StaticFileHandler',
+  extends: 'foam.net.node.Handler',
+
+  imports: [
+    'log',
+    'info',
+    'warn'
+  ],
+
+  properties: [
+    {
+      class: 'String',
+      name: 'dir',
+      documentation: 'Directory under which to serve files.',
+      preSet: function(old, nu) {
+        return this.path.resolve(process.cwd(), nu);
+      },
+      factory: function() { return process.cwd(); }
+    },
+    {
+      class: 'String',
+      name: 'urlPath',
+      documentation: 'URL path prefix. Stripped before searching "dir".'
+    },
+    {
+      name: 'mimeTypes',
```

Comment:
Isn't this duplicated from above? Could it be a trait or in a LIB? Is this mutated? Could it be a constant instead?